readthedocs/rtd_tests/tests/test_doc_builder.py

```diff
@@ -36,12 +36,9 @@ def setUp(self):
         BaseSphinx.type = 'base'
         BaseSphinx.sphinx_build_dir = tempfile.mkdtemp()
 
-    # FIXME DOCSITALIA: This test currently hits the api due to docsitalia.utils.get_subprojects
     @patch('readthedocs.doc_builder.backends.sphinx.BaseSphinx.docs_dir')
     @patch('readthedocs.projects.models.Project.checkout_path')
     @override_settings(DONT_HIT_API=True)
-    @pytest.mark.skip('This test currently hits the api due to docsitalia.utils.get_subprojects')
-    @pytest.mark.failing
     def test_conf_py_path(self, checkout_path, docs_dir):
```

Comment:
how about adding `@patch('readthedocs.docsitalia.utils.get_subprojects')` to skip the api call entirely?

Reply:
I'm ok with your proposal